### ISSUE TITLE
Drush php7 container is an issue for sites with incompatible php syntax

### ISSUE THREAD

**Issue by AubreyR** [kalabox/kalabox#1369]
Bug Report
==========

**Tell us about your setup**

What is your Kalabox version and operating system?

*0.12.0-beta.1 on Ubuntu 16.04 (kernel: 4.4.0-22-generic)*

Have you previously installed Kalabox? If so, what version?

*Yes, I've installed Kalabox v0.12.0-alpha3 - Never got it working though*

**Tell us about your bug**

Give a brief description of what happened, including any error messages that appeared.

*I tried to create a pantheon site but whenever I try to install the app, the drush command throws a failure at this point.  It throws a PHP Fatal Error near the end of the drupal setup.  This error corresponds to an error that should not be thrown with PHP 5.5 (this type of error, having a break statement outside a loop or switch is only a fatal error since PHP 7).  Everyone else in my office has the same error upon clean install.  The error does not appear when creating normal drupal sites, only when we create/clone a Pantheon app.*

![phperror](https://cloud.githubusercontent.com/assets/5516067/15514567/7f8413c8-219e-11e6-951e-007b5d3e172c.png)

Please detail the steps we can take to replicate this bug. Be as detailed as possible.

> 1. Create a Pantheon site with Drupal 7
> 2. Include modules in the site which have php "break" keywords outside of loop or switch statements.
> 3. Using kbox cli, create a local app from the pantheon site.

**Tell us more**

*App Kalabox.yml output (truncated the repo info at the end):*

![kalaboxyml](https://cloud.githubusercontent.com/assets/5516067/15514657/f9112e38-219e-11e6-9755-73a45aedbc83.png)

Does this bug prevent you from using Kalabox?

*Yes, as I must install an older alpha version if I want to attempt to use it (which I could never get to work)*
**EDIT:** *I cannot log into the drupal admin dashboard as when I attempt to visit /user a blank page is displayed*

**Comment by pirog** [comment]
@AubreyR does this actually stop you from developing on this site? 

It looks like this is just happening when drush tries to rebuild your registry. Guessing the drush container runs php7 but your webserver should be php 5.5

**Comment by AubreyR** [comment]
@pirog That's what I thought as well.  So how to I rectify this?  I have no idea about the underlying do-dads of Kalabox.  Isn't it up to Kalabox to determine which version of PHP should be running on the drush container?

**Comment by leevh** [comment]
@pirog  Yes, it is stopping us from developing on sites created with beta1.  The site will initially appear fine on first visit, but if we try to visit /user to log in, we receive a blank page.  With the same symptoms as @AubreyR, I've checked the webserver and confirmed its 5.5, and also that the kbox CLI is php7, as you suggested.  Could the CLI container php version have some effect on the site install if drush is throwing errors?

**Comment by leevh** [comment]
Just tested a fresh install of drupal 7 and 8 and both work perfectly.  It's just our sites from pantheon that won't work on beta1.  I presume any pantheon site that contains code that would cause drush to complain about php7 related errors will have this problem

**Comment by pirog** [comment]
I definitely think we need to move the drush container to run on php 5.5 that said the drush container should not impact whether your site loads or not.

**Comment by AubreyR** [comment]
Same symptom as @leevh if I attempt to visit the /user it just displays a blank page.  This behaviour is exactly the same on all of my co-workers computers (that's three to be exact).

**Comment by leevh** [comment]
@pirog agreed, but it must be drush registry rebuild / cache clearing that is supposed to happen after the DB is pulled.  The drush errors don't allow this to complete and the DB pull is somehow incomplete.  

As an update, I've edited kalabox-cli.yml for the faulty app and on line 57/62 I've set it to build from local dockerfile for terminus.  In that dockerfile I set "FROM drush/drush:8-php5".   Did a kbox rebuild and kbox pull and the site is now functional
